loleaflet/html/loleaflet.html.m4

```diff
@@ -202,6 +202,11 @@ m4_ifelse(MOBILEAPP,[true],
     <input id="insertgraphic" type="file" accept="image/*" style="position: fixed; top: -100em">
     <input id="selectbackground" type="file" accept="image/*" style="position: fixed; top: -100em">
 
+    <div class="userInfo">
+      <div class="userInfo-content" id="listActiveUser">
+      </div>
+    </div>
```

Comment:
Let's just use one unique id here for those elements as the class `userInfo` is not really unambiguously. See the other comment for the naming in general as well :wink: 

It could be something like this:
```suggestion
    <div id="userListHeader">
      <div id="userListSummary"></div>
      <div id="userListPopover"></div>
    </div>
```